### DISCUSSION TITLE
A "copy response" button.

### DISCUSSION THREAD

**Discussion by jonasfroeller** [scalar/scalar#1270] [Ideas]
I would like to copy or download the response body as a file.

**Comment by marclave** [comment]
@jonasfroeller this is a great idea!

@hanspagel @amritk we can add a copy button in the api client for the response ✨

**Comment by FredRobehmed-Alight** [comment]
My team could also use this feature. With how the body is lazily rendered (great for performance), we aren't able to copy our responses out from the browser client - so we have to either use another tool or pull the response out of the browser's dev tools.
